apps/src/codebridge/utils/validateFileName.ts

```diff
@@ -23,15 +23,16 @@ export const validateFileName = ({
   if (!fileName.length) {
     return;
   }
-  if (!isValidFileName(fileName)) {
-    return codebridgeI18n.invalidNameError();
-  }
 
   const [, extension] = fileName.split('.');
   if (!extension) {
     return codebridgeI18n.noFileExtensionError();
   }
 
+  if (!isValidFileName(fileName)) {
+    return codebridgeI18n.invalidNameError();
+  }
+
```

Comment:
This addresses the report from the bug bash that had questioned a file named `Pandas`. Luckily I'd bumped into the same thing earlier today and knew exactly what it was. :-) 

So it just moves the no file extension check a little earlier.

Reply:
It is a bit distracting to see a red error message when you haven't yet finished writing the file name. Should we delay showing this error message until the user tries to click ok?